src/components/Books/BooksList.js

```diff
@@ -9,15 +9,26 @@ const BooksList = () => {
     dispatch(removeExistBook(id));
   };
   useEffect(() => dispatch(loadAndDisplayBooks()), []);
+  let i = 0;
   return (
-    <ul>
+    <ul className="books" style={{ gridTemplateRows: `repeat(${books.length},1fr)` }}>
       {books.map((book) => {
-        const { id, title, author } = book;
+        const {
+          id, title, author, category,
+        } = book;
+        i += 1;
         return (
-          <li key={id}>
-            <h2>{title}</h2>
-            <p>{author}</p>
-            <button type="submit" onClick={() => handleRemove(id)}>Remove</button>
+          <li key={id} className="book-card" style={{ gridRow: `${i}/span 1` }}>
+            <div className="book-inf">
+              <p className="category">{category}</p>
+              <h2>{title}</h2>
+              <p className="author">{author}</p>
+              <div className="user-btns">
+                <button className="edit-button" type="submit">Comments</button>
+                <button className="edit-button" type="submit" onClick={() => handleRemove(id)}>Remove</button>
+                <button className="edit-button" type="submit">Edit</button>
+              </div>
+            </div>
           </li>
```

Comment:
- [ ] As the requirements show: 😉 

![design-requirements](https://user-images.githubusercontent.com/79876959/145491491-f7d9c9a4-c4aa-4efb-8d93-77913e556283.PNG)

It is necessary to implement this part too for each card. (You don't need to add any functionality for those.)

![design-requirements-2](https://user-images.githubusercontent.com/79876959/145491681-b6642b3b-606b-4d42-8d2d-0525020ddee0.PNG)